src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -136,6 +139,8 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
 
         this.chartType = this.props.chartType;
 
+        this.colorMap = new Map();
```

Comment:
@alisman what's our take on the Map vs Object in the project? I barely used Map, is it something we can mix?